File: q431/helloworld.cs
using System;
class HelloWorld{
	static void Main(){
		int szero=(new int[]{}).Length.ToString().Length;
		//int szero=System.Convert.ToInt32(true);
		int sone=szero<<szero;
		int stwo=szero<<(szero+szero);
		int sthree=szero<<(szero+szero+szero);
		int sfour=szero<<(szero+szero+szero+szero);
		int sfive=szero<<(szero+szero+szero+szero+szero);
		int ssix=szero<<(szero+szero+szero+szero+szero+szero);

		Console.Write((char)(ssix|sthree));
		Console.Write((char)(ssix|sfive|stwo|szero));
		Console.Write((char)(ssix|sfive|sthree|stwo));
		Console.Write((char)(ssix|sfive|sthree|stwo));
		Console.Write((char)(ssix|sfive|sthree|stwo|sone|szero));
		Console.Write((char)(sfive));
		Console.Write((char)(ssix|sfour|stwo|sone|szero));
		Console.Write((char)(ssix|sfive|sthree|stwo|sone|szero));
		Console.Write((char)(ssix|sfive|sfour|sone));
		Console.Write((char)(ssix|sfive|sthree|stwo));
		Console.Write((char)(ssix|sfive|stwo));
		Console.WriteLine();
	}
}

File: q431/helloworld.vb
module HelloWorld
	sub Main(ByVal args() as String)
		dim szero,sone,stwo,sthree,sfour,sfive,ssix as integer
		szero=(new integer(){}).Length.ToString().Length
		sone=szero<<szero
		stwo=szero<<(szero+szero)
		sthree=szero<<(szero+szero+szero)
		sfour=szero<<(szero+szero+szero+szero)
		sfive=szero<<(szero+szero+szero+szero+szero)
		ssix=szero<<(szero+szero+szero+szero+szero+szero)

		Console.Write(Chr(ssix+sthree))
		Console.Write(Chr(ssix+sfive+stwo+szero))
		Console.Write(Chr(ssix+sfive+sthree+stwo))
		Console.Write(Chr(ssix+sfive+sthree+stwo))
		Console.Write(Chr(ssix+sfive+sthree+stwo+sone+szero))
		Console.Write(Chr(sfive))
		Console.Write(Chr(ssix+sfour+stwo+sone+szero))
		Console.Write(Chr(ssix+sfive+sthree+stwo+sone+szero))
		Console.Write(Chr(ssix+sfive+sfour+sone))
		Console.Write(Chr(ssix+sfive+sthree+stwo))
		Console.Write(Chr(ssix+sfive+stwo))
		Console.WriteLine()
	end sub
end module